aicsimageio/readers/czi_reader.py

```diff
@@ -751,6 +754,13 @@ def _get_stitched_dask_mosaic(self) -> xr.DataArray:
     def _get_stitched_mosaic(self) -> xr.DataArray:
         return self._construct_mosaic_xarray(self.data)
 
+    @property
+    def ome_metadata(self) -> OME:
+        return metadata_utils.transform_metadata_with_xslt(
+            self.metadata,
+            Path(__file__).parent.parent / "metadata/czi-to-ome-xslt/xslt/czi-to-ome.xsl"
```

Comment:
This should be split into:

```python
"metadata" / "czi-to-ome-xslt" / "xslt" / "czi-to-ome.xsl"
```

So that we aren't assuming the OS path separator is `/`.

Reply:
Ah good call, thanks!

Reply:
@JacksonMaxfield I actually don't think this is necessary. From the googling I did, `pathlib` will fix the slashes for you even if you have them in a string like I do. This is backed up by the fact that the Windows tests are passing.